resolwe_bio/processes/support_processors/mutations_table.py

```diff
@@ -498,3 +536,62 @@ def run(self, inputs, outputs):
         outputs.genes = genes
         outputs.species = inputs.variants.output.species
         outputs.build = inputs.variants.output.build
+
+        # Store the reported variants in the Variant Database
+        data_source = "RNA-Seq Variant Calling Pipeline"
+
+        output_table["species"] = inputs.variants.output.species
+        output_table["genome_assembly"] = (
+            "GRCh38"
+            if "GRCh38" in inputs.variants.output.build
+            else inputs.variants.output.build
+        )
+
+        sample_fields_map = {
+            "AD": "SAMPLENAME1.AD",
+            "GT": "SAMPLENAME1.GT",
+            "GQ": "SAMPLENAME1.GQ",
+        }
+
+        columns_map = {
+            "species": "species",
+            "genome_assembly": "genome_assembly",
+            "CHROM": "chromosome",
+            "POS": "position",
+            "REF": "reference",
+            "ALT": "alternative",
+        }
+
+        optional_mapping = {
+            "QUAL": "quality",
+            "DP": "depth",
+            "FILTER": "filter",
+            "QD": "depth_norm_quality",
+            "SAMPLENAME1.AD": "unfiltered_allele_depth",
+            "SAMPLENAME1.GT": "genotype",
+            "SAMPLENAME1.GQ": "genotype_quality",
+        }
+
+        requested_fields = inputs.vcf_fields + inputs.advanced.gf_fields
+        requested_fields = [
+            sample_fields_map.get(field, field) for field in requested_fields
+        ]
+
+        for key, value in optional_mapping.items():
+            if key in requested_fields:
+                columns_map[key] = value
+
+        output_table = output_table.rename(columns=columns_map)
+
+        if "genotype" in output_table.columns:
+            output_table["genotype"] = output_table.apply(
+                encode_variant_genotype, axis=1, result_type="reduce"
+            )
+
+        if "unfiltered_allele_depth" in output_table.columns:
+            output_table["unfiltered_allele_depth"] = output_table.apply(
```

Comment:
This step could be vectorised to speed up the processing `output_table["unfiltered_allele_depth"].str.split(",").str[1].astype(int)`.